plugin-api.d.ts

```diff
@@ -1016,16 +1001,8 @@ interface ChildrenMixin {
   findChildren(callback?: (node: SceneNode) => boolean): SceneNode[]
   findChild(callback: (node: SceneNode) => boolean): SceneNode | null
 
-  /**
-   * If you only need to search immediate children, it is much faster
-   * to call node.children.filter(callback) or node.findChildren(callback)
-   */
   findAll(callback?: (node: SceneNode) => boolean): SceneNode[]
 
-  /**
-   * If you only need to search immediate children, it is much faster
-   * to call node.children.find(callback) or node.findChild(callback)
-   */
   findOne(callback: (node: SceneNode) => boolean): SceneNode | null
```

Comment:
I have a separate PR to update the docs for:
findAll https://www.figma.com/plugin-docs/api/properties/nodes-findall/
findOne https://www.figma.com/plugin-docs/api/properties/nodes-findone/